eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs

```diff
@@ -144,6 +144,10 @@ instance Typeable era => ToCBOR (RedeemersRaw era) where
 instance Memoized Redeemers where
   type RawType Redeemers = RedeemersRaw
 
+-- | Note that 'Redeemers' are based on 'MemoBytes' since we must preserve
+-- the original bytes for the 'ScriptIntegrity'.
```

Comment:
In order for haddock to properly link it
```suggestion
-- the original bytes for the 'Cardano.Ledger.Alonzo.Tx.ScriptIntegrity'.
```

Reply:
thanks!